backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -26,11 +56,24 @@ module.exports = ({ prisonApi, logError }) => async (req, res) => {
     ])
 
     const userCaseLoadIds = userCaseLoads.map(caseLoad => caseLoad.caseLoadId)
-
     const { bookingId, firstName, lastName } = prisonerDetails
-
     const currentPrisonerDetails = locationHistory.find(record => record.bookingId === bookingId) || {}
 
+    const { movementMadeBy, assignmentReason, bedAssignmentHistorySequence } = currentPrisonerDetails
+    const movementMadeByName = await fetchStaffName(res.locals, movementMadeBy, prisonApi)
+    const assignmentReasonName = assignmentReason
+      ? await fetchReasonDescription(res.locals, assignmentReason, caseNotesApi)
```

Comment:
Don't fix now but when for next time. You could simplify this by doing:
`
    const assignmentReasonName = assignmentReason && await fetchReasonDescription(res.locals, assignmentReason, caseNotesApi)
     
`